src/PIL/ImageFile.py

```diff
@@ -40,6 +40,7 @@
 SAFEBLOCK = 1024 * 1024
 
 LOAD_TRUNCATED_IMAGES = False
+"""Whether loading of truncated image files is currently enabled. User code may change this."""
```

Comment:
```suggestion
"""Whether or not to load truncated image files. User code may change this."""
```
Fixes the lint error that the line is too long. You could also break your text into two lines, but here's my suggestion.